chore(docs): moving prerequisites (#2475)
winglang/wing: docs/02-getting-started/05-aws.md

:::

## Prerequisites

> To deploy Wing code to AWS, you'll need the following setup:

Reviewer: ```suggestion
In order to deploy to AWS, you will need:
```
